docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/create-hybrid-node-pools.md

```diff
@@ -0,0 +1,370 @@
+    nodes during the [Add CNI Cluster Profile](./import-eks-cluster-enable-hybrid-mode.md#add-cni-cluster-profile)
+    steps.
+
+    :::
+
+15. Click **Confirm** when complete.
+
+16. In **Profile Layers**, click **Next** to continue.
+
+17. Click **Finish Configuration**.
+
+Your cluster profile for hybrid nodes is now created and can be used in the
+[Create Hybrid Node Pool](#create-hybrid-node-pool) steps.
+
+### Validate
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu**, select **Profiles**.
+
+3. On the **Profiles** page, click on your newly created profile.
+
+4. Check that your profile layers are present and correct.
+
+## Create Hybrid Node Pool
+
+### Prerequisites
+
+- An Amazon EKS cluster imported with hybrid mode enabled. Refer to
+  [Import EKS Cluster and Enable Hybrid Mode](./import-eks-cluster-enable-hybrid-mode.md) for guidance.
+
+- Edge hosts have been registered with Palette through
+  [Agent Mode](../../../../deployment-modes/agent-mode/agent-mode.md) or by using
+  [provider images](../../../edge/edgeforge-workflow/palette-canvos/build-provider-images.md).
+
+  :::warning
+
+  If using provider images, you must include the following in your `.arg` file during the
+  [build steps](../../../edge/edgeforge-workflow/palette-canvos/build-provider-images.md#build-provider-images).
+
+  ```shell
+  K8S_DISTRIBUTION=nodeadm
+  K8S_VERSION=<kubernetesVersion>  # supported versions: [ 1.28.0 | 1.29.0 | 1.30.0 | 1.31.0 ]
+  ```
+
+  Replace `<kubernetesVersion>` with your version of Kubernetes. For example, `1.29.0`.
+
+  :::
+
+- A cluster profile created for your hybrid nodes. Refer to
+  [Create Cluster Profile for Hybrid Node Pools](#create-cluster-profile-for-hybrid-node-pools) for steps.
+
+### Create Node Pool
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu**, select **Clusters**.
+
+3. Select your cluster to view its **Overview** tab.
+
+4. Select the **Nodes** tab.
+
+5. In the top-right, click **New Node Pool**.
+
+6. In the pop-up window, fill in the following fields.
+
+   | **Field**                        | **Description**                                                                                                                                                                                                                 |
+   | -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | **Node pool name**               | A descriptive name for the hybrid node pool.                                                                                                                                                                                    |
```

Comment:
I personally prefer that fields, even if they are part of a table, be bolded, but I noticed they aren't always. This is the first time I've actually seen them bolded in the docs I've looked at so far, so I'm just drawing attention to it.

Reply:
In this context, I'm bolding them because they are product UI fields, which meets the criteria according to our style guide.